infrastructure/scheduler/cloudwatch.tf

```diff
@@ -2,18 +2,18 @@ resource "random_integer" "time_delay" {
   min   = 0
   max   = 59
   seed  = "${var.scan_hosts[count.index]}"
-  count = "${length(var.scan_hosts)}"
+  count = "${(length(var.scan_hosts)>80)?80:length(var.scan_hosts)}"
 }
 
 resource "aws_cloudwatch_event_target" "task_scheduler" {
-  count = "${length(var.scan_hosts)}"
+  count = "${(length(var.scan_hosts)>80)?80:length(var.scan_hosts)}"
   rule  = "${aws_cloudwatch_event_rule.task_scheduler_sqs_rule.*.name[count.index]}"
   arn   = "${var.queue_arn}"
-  input = "{\"CloudWatchEventHost\":\"${var.scan_hosts[count.index]}\"}"
+  input = "{\"CloudWatchEventHosts\":[\"${join("\",\"",slice(var.scan_hosts, floor(count.index*(length(var.scan_hosts)/80)), ceil((count.index+1)*(length(var.scan_hosts)/80)) ) )}\"]}"
```

Comment:
have we checked that you can do this when the number of hosts isn't a multiple of 80? Do we need a min of the 80*(1+index) and length(list)?

Reply:
good point and something I forgot to work through before the PR - all fixed and tested with edge cases covered. I want to leave a note here that CWE rules are per account (not user) so we may need to change this since the task runs against an account...